fcrepo-stats-api/src/main/java/org/fcrepo/stats/api/StatsParameters.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.stats.api;
+
+import java.time.Instant;
+import java.util.List;
+
+/**
+ * A parameter object that allows you to narrow your statistics query
+ * @author dbernstein
+ */
+public class StatsParameters {
+
+    private List<String> mimeTypes;
+
+    private List<String> rdfTypes;
+
+    private Instant createdAfter;
+
+    private Instant updatedBefore;
+
+    /**
+     * Constructor
+     * @param mimeTypes Include only specified mimetypes types in stats count (null ok)
+     * @param rdfTypes Include only specified RDF types in stats count (null ok)
```

Comment:
None of these parameters are used or implemented besides mimeTypes. I'd lean towards removing them unless there is a strong expectation that they will definitely be added in the near future, to avoid potential confusion later.